lib/topics/state_and_behaviour.rb

```diff
@@ -2,4 +2,31 @@
 
 # exercise state and behaviour
 module StateAndBehaviour
+  # class Car / Default car "Chargo"
+  class Car
+    attr_accessor :color, :year, :model, :current_speed
+
+    def initialize(params)
+      @color = params[:color] || 'red'
+      @model = params[:model] || 'Chargo'
+      @year = params[:year] || 1981
+      @current_speed = 0
```

Comment:
when I use constants rubocop an error "dynamic constant assignment"
      START_SPEED = 0
      DEFAULT_COLOR = "red"
      DEFAULT_MODEL = "Chargo"
      DEFAULT_YEAR = 1981
Why? And how can I fix error?